decidim-admin/app/views/layouts/decidim/admin/_sidebar.html.erb

```diff
@@ -8,6 +8,9 @@
 
 <nav class="main-menu">
   <%= active_link_to t("menu.dashboard", scope: "decidim.admin"), root_path, active: :exact %>
+  <% if can? :read, current_organization %>
```

Comment:
Do you think you could put this on the last place of the menu? It'd feel way more natural to me. THAAAANKS